.github/workflows/publish.yml

```
@@ -70,3 +70,12 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      # Update Readme on Docker Hub
+      - name: Docker Hub Description
+        if: github.event_name != 'pull_request'
+        uses: peter-evans/dockerhub-description@v3
```

Pin to specific version to have reproducible builds. Renovate will taking care of updating actions afterwards.

```suggestion
        uses: peter-evans/dockerhub-description@14881160433a81f9ebc838a253d4edd9e8fc35f1 # tag=v3.1.1
```